app/docker/networks/edit/NetworkOptionsTable.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { render } from '@/react-tools/test-utils';
+
+import { NetworkOptions } from '../types';
+
+import { NetworkOptionsTable } from './NetworkOptionsTable';
+
+const options: NetworkOptions = {
+  'com.docker.network.bridge.default_bridge': 'true',
+  'com.docker.network.bridge.enable_icc': 'true',
+  'com.docker.network.bridge.enable_ip_masquerade': 'true',
+  'com.docker.network.bridge.host_binding_ipv4': '0.0.0.0',
+  'com.docker.network.bridge.name': 'docker0',
+  'com.docker.network.driver.mtu': '1500',
+};
+
+test('Network options values should be visible', async () => {
+  const { findByText, findAllByText } = render(
+    <NetworkOptionsTable options={options} />
+  );
+
+  await expect(findByText('Network options')).resolves.toBeVisible();
+  // expect to find three 'true' values for the first 3 options
+  const cells = await findAllByText('true');
+  expect(cells[0]).toBeVisible();
+  expect(cells[1]).toBeVisible();
+  expect(cells[2]).toBeVisible();
```

Comment:
findAll will only return the cells that exist. if you want to check they exist, just do `expect(cells).toHaveLength(3)`

Reply:
updated